### PR TITLE
Initial build for frontend

### PR DESCRIPTION
**Pull request by Rec0iL99**
**Changes**

* All screens proposed in mockup designed in XML.
* Connection with API and frontend implemented.
* Basic functionality for frontend implemented.
* Square android codestyle installed for the repo.

**App demo in Appetize** - [see here](https://appetize.io/app/tp8ad8qz0vewjgbgtepc1d3r98?device=nexus5&scale=75&orientation=portrait&osVersion=8.1)

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
```
Comment (jeffin143) [inline .idea/codeStyles/codeStyleConfig.xml:3]: Thanks for this :)

File: app/src/test/java/com/khacks/kapp_frontend/ExampleUnitTest.kt [hunk 2]
```diff
@@ -10,8 +10,8 @@ import org.junit.Assert.*
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
-    @Test
-    fun addition_isCorrect() {
-        assertEquals(4, 2 + 2)
-    }
+  @Test
+  fun addition_isCorrect() {
+    assertEquals(4, 2 + 2)
```
Comment (jeffin143) [inline app/src/test/java/com/khacks/kapp_frontend/ExampleUnitTest.kt:15]: This is irrelevant right ?

**Review by jeffin143 — APPROVED** [review]
Took a over all look and it looks ok to me, I will go ahead and merge it and if I find a issue, will open a issue for you

File: app/src/test/java/com/khacks/kapp_frontend/ExampleUnitTest.kt (hunk 2 above)
Comment (Rec0iL99) [reply app/src/test/java/com/khacks/kapp_frontend/ExampleUnitTest.kt:15]: It is an generated example test

**Pull request merged**